### DISCUSSION TITLE
Add label and "Open in new window" for native link fields

### DISCUSSION THREAD

**Discussion by maxstrebel** [engram-design/FeedMe#1559] [Ideas]
Hey, I am very happy that we finally have a first party Link Field. I will oftentimes consume feeds with feed-me and today I realized that I cannot set label and "Open in new window". It would be great if we could fill all native options. Would that be possible?

**Field Options**
<img width="1029" alt="image" src="https://github.com/user-attachments/assets/4fe75008-8a8f-42b6-a606-84a6695ebcb0">

**Import Options**
<img width="995" alt="image" src="https://github.com/user-attachments/assets/44679c59-968f-4775-a9dd-b0770db17c80">

**Comment by AmarHuda1** [comment]
Yeah, I also hope you can add the Link Type when importing a Link Field. When we intend to import an Asset Link into the Link Field, we would like the imported content to select the Asset, rather than applying the URL to the link type URL. This way, in the import JSON file, we can pass the url alongside the type we want.
```
{
   "type": "url",
   "url": "http://google.com"
},
{
   "type": "entries",
   "url": "Entry URL"
},
{
   "type": "asset",
   "url": "Asset Url"
},
{
   "type": "email",
   "url": "test@example.com"
}
```
